examples/registry_example/README.md

```diff
@@ -0,0 +1 @@
+# registry Example
```

Comment:
Lets provide more details here on what this is meant showcase and any relevant files / code.